src/code/z_camera.c

```diff
@@ -7020,10 +7020,19 @@ void Camera_PrintSettings(Camera* camera) {
         if (camera->camDataIdx < 0) {
             sp50[i++] = '-';
         }
+
+        // @bug: this code was clearly meaning to print camera->camDataIdx as a two-digit
+        // number, not i.
```

Comment:
```suggestion
        //! @bug: this code was clearly meaning to print `camera->camDataIdx` as a two-digit
        //! number, instead of `i`.
```
I think this is the right way to format bug comments?

also this is probably not the only thing wrong, there's a check for `camera->camDataIdx < 0` but I don't think the following code would work correctly with a negative number?